config/rbac/kustomization.yaml

```diff
@@ -13,6 +13,8 @@ resources:
 # default, aiding admins in cluster management. Those roles are
 # not used by the Project itself. You can comment the following lines
 # if you do not want those helpers be installed with your Project.
+- operator_vlogs_editor_role.yaml
```

Comment:
looks like it should disabled by default

Reply:
it was removed automatically by kubebuilder, looks like it doesn't like config folder structure